### PR TITLE
Issue/lp 7

### PR DESCRIPTION
**Pull request by lc-thomasberger**
handle languagePicker:languageChange Event

### CODE REVIEW

**Comment by lc-thomasberger** [comment]
#43

File: js/adapt-contrib-resources.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ define([
 
     }
 
-    Adapt.once('app:dataReady', function() {
+    function initRessources() {
```
Comment (moloko) [inline js/adapt-contrib-resources.js]: typo - should be `initResources`

**Comment by moloko** [comment]
whilst you're working on this could you remove the commented out lines 34 & 35?

**Comment by moloko** [comment]
Maybe I missed this before... but if I have a course in English and German and, on first launch, I choose German I get a double resources in the drawer:
2016-09-28 15_48_04-de-_ adapt version 2 0 demonstration

If I close and relaunch it works properly again.

If I switch languages mid-course using the 'picker' it all works fine.

**Comment by moloko** [comment]
I think I have a solution for the above, will push shortly

**Comment by moloko** [comment]
what am I doing? I can't approve this as I committed code to it - sorry!

**Comment by moloko** [comment]
@tomgreenfield could you review this?

**Comment by tomgreenfield** [comment]
Personally I still think it's easier to read if we listened to `app:dataReady` multiple times (changing `.once` → `.on`) than having multiple listeners set up. If the functionality differed on a change of language then I would be persauded otherwise.

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**